src/layout/UserMenuLoginForm.vue

```diff
@@ -0,0 +1,222 @@
+<template>
+  <div>
+    <v-form @submit.prevent="login" class="my-8" v-if="!signUpMode">
+      <v-text-field
+        v-if="!isDomainLocked"
+        v-model="domain"
+        name="domain"
+        label="Girder Domain"
+        required
+        prepend-icon="mdi-domain"
+      />
+      <div id="login-embed" v-if="outsetaLoginDomain"></div>
+      <template v-else>
+        <v-text-field
+          v-model="username"
+          name="username"
+          label="Username or e-mail"
+          required
+          prepend-icon="mdi-account"
+          autocomplete="username"
+        />
+        <v-text-field
```

Comment:
Should we have a "re-enter password" field as well to catch mistakes?

Reply:
I added one, but it's pretty crude, it just disables the sign up button if the password and password verification fields don't match  
I can improve it if necessary, but it should probably go it another PR